src/mozillavpn.cpp

```diff
@@ -1216,7 +1216,9 @@ void MozillaVPN::backendServiceRestore() {
 void MozillaVPN::heartbeatCompleted(bool success) {
   logger.debug() << "Server-side check done:" << success;
 
-  if (!success) {
+  // In the event of a Guardian error during authentication,
+  // the "something went wrong" screen is displayed to the user.
```

Comment:
Should the comment explain why (because the service that is used for authentication is unavialble)? Should the comment also say that other heartbeat errors are ignored to prevent unnecessary VPN interruptions?

Reply:
Definitely worth adding a bit more context!